WordPress/Classes/ViewRelated/Blog/QuickStartNoticeView.swift

```diff
@@ -1,9 +1,147 @@
 class QuickStartNoticeView: NoticeView {
-//    override enum Appearance {
-//        static let actionBackgroundColor = UIColor.black.withAlphaComponent(0.5)
-//        static let shadowColor: UIColor = .black
-//        static let shadowOpacity: Float = 0.25
-//        static let shadowRadius: CGFloat = 8.0
-//        static let shadowOffset = CGSize(width: 0.0, height: 2.0)
-//    }
+    private let yesButton = UIButton(type: .system)
+    private let noButton = UIButton(type: .system)
+
+    override func configure() {
+        configureBackgroundViews()
+        configureShadow()
+        configureContentStackView()
+        configureLabels()
+        configureForNotice()
+    }
+
+    static func makeHighlightMessage(base normalString: String, highlight: String, icon: UIImage) -> NSAttributedString {
```

Comment:
I think this could probably move to a private `String` or `NSAttributedString` extension.